nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
 ---
-title: Microsoft Teams JavaScript client library
+title: Microsoft Teams Client API reference
 description: Reference documentation for the latest Microsoft Teams JavaScript client library
-keywords: msteams teams client sdk javascript library reference latest
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams Client API reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This page contains general information about the client library to support you in the development of using The Microsoft Teams Client SDK.
```

Comment:
Also generalize the description here (Teams Client SDK implies only TeamsJS, as that's what it used to be called)

Reply:
I feel like at some point I heard guidance to say article vs page. So maybe do something like "This article provides you with general information about the libraries that Microsoft offers to support you in the development of..."